.github/workflows/CI.yml

```diff
@@ -704,6 +704,53 @@ jobs:
           cd testing-framework/terraform
           make terraformCleanup
 
+  run-logs-testbed:
```

Comment:
You also need to update the release candidate and publish status job to have a dependency (need) on this new job

Reply:
it seems that this comment was addressed.